docs/multi-cluster-service.md

```diff
@@ -6,7 +6,7 @@ Users are able to **export** and **import** services between clusters with [Mult
 
 ### Karmada has been installed
 
-We can install Karmada by referring to [quick-start](https://github.com/karmada-io/karmada#quick-start), or directly run `hack/local-up-karmada.sh` script which is also used to run our E2E cases.
+We can install Karmada by referring to [Quick Start](https://github.com/karmada-io/karmada#quick-start), or directly run `hack/local-up-karmada.sh` script which is also used to run our e2e cases.
```

Comment:
Is there a requirement for the case of `e2e`?

Reply:
I understand the description here to mean that you can use 'hack/ local-up-karmda.sh' to deploy Karmada, and this script was also used in e2e testing

Reply:
Sorry, I didn't make it clear, I was actually referring to the case format.

Reply:
OK, do you mean keep E2E?